force-app/main/default/objects/HOT_Request__c/validationRules/NumericPhone.validationRule-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>NumericPhone</fullName>
+    <active>true</active>
+    <errorConditionFormula>!ISBLANK(OrdererPhone__c)
+&amp;&amp;
+(
+(LEN(OrdererPhone__c) &lt; 8 || LEN(OrdererPhone__c) &gt; 14)
+||
+!ISNUMBER(OrdererPhone__c)
+||
+BEGINS(OrdererPhone__c, &apos;-&apos;)
```

Comment:
Sjekker du om den startet på "+", ser ut som det står minus?

Reply:
om nummeret starter med minus- eller pluss-tegn, tolkes det som numeric. ISNUMBER() blir da true.
For å forhindre at de skriver minus-tegn, men godta pluss-tegn, er dette den beste måten å gjøre det på.
Hvis ikke måtte vi sjekka ! BEGINS( '+' ) && ! ISNUMBER(LEFT(field, 1)). Føler det var lettere å forhindre minus, istedenfor å godta pluss og numeric som første "siffer".

Reply:
Den er grei.